File: src/main/java/org/amithalpert/FlappyNix.java
package org.amithalpert;
import java.awt.*;


public class FlappyNix extends Thread {
	Player player;
	Rectangle groundCollider = new Rectangle(700, 900, 1200, 1000);
	GameScreen panel;

	public FlappyNix(Player player, GameScreen p) {
		this.player = player;
		this.panel = p;
		start();
	}


	public void draw(Graphics g) {

	}


	public void run() {


		while (true) {


			if (groundCollider.intersects(player.hitBox)) {
				System.out.println("GG");
			}


		}

	}




}

File: src/main/kotlin/FlappyNix.kt
import java.awt.Color
import java.awt.Graphics
import kotlin.math.pow
import kotlin.math.sqrt

class FlappyNix(var x: Int, var y: Int, var width: Int, var color: Color, var panel: GameScreen) : Thread() {
    fun draw(g: Graphics) {
        g.color = color
        g.fillOval(x - width / 2, y - width / 2, width, width)
    }

    override fun run() {
        var dirx = 6
        while (true) {
                x += dirx


                try {
                    sleep(7)
                } catch (e: InterruptedException) {
                }

        }
    }
}
